plugin/core/main.py

```diff
@@ -153,3 +155,12 @@ def on_new_window_async(self, w: sublime.Window) -> None:
 
     def on_pre_close_window(self, w: sublime.Window) -> None:
         windows.discard(w)
+
+    def on_load(self, view: sublime.View) -> None:
+        file_name = view.file_name()
+        if not file_name:
+            return
+        for fn, tup in opening_files.items():
+            if fn == file_name or os.path.samefile(fn, file_name):
+                opening_files.pop(fn)
+                return tup[1](view)
```

Comment:
Of course, it's not a problem in practice but I'd just `break` on a separate line instead to make intentions clearer.